prediction_market_agent_tooling/tools/relevant_news_analysis/relevant_news_cache.py

```
@@ -0,0 +1,87 @@
+        with Session(self.engine) as session:
+            query = (
+                select(RelevantNewsCacheModel)
+                .where(RelevantNewsCacheModel.question == question)
+                .where(RelevantNewsCacheModel.days_ago <= days_ago)
+                .where(RelevantNewsCacheModel.datetime_ >= utcnow() - timedelta(days=1))
+            )
+            item = session.exec(
+                query.order_by(desc(RelevantNewsCacheModel.datetime_))
+            ).first()
+
+            if item == None:
```

Ruff

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)